test/mocha-test.el

```diff
@@ -0,0 +1,11 @@
+
+;;;; mocha-find-project-root
+
+(ert-deftest mocha-test/mocha-find-project-root/return-path-to-project-root ()
+  (mocha-test/with-sandbox
+   (should (f-same? (mocha-find-project-root) mocha-test/sandbox-path))))
+
+(ert-deftest mocha-test/mocha-find-project-root/return-nil-unless-package-file-exist ()
```

Comment:
Can we add another test where we create a subdirectory in the sandbox, switch into it, and then make sure the project root is still the sandbox dir?